mlir/include/mlir/Dialect/MQTOpt/Transforms/Transpilation/Unit.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+ * Copyright (c) 2025 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#pragma once
+
+#include "mlir/Dialect/MQTOpt/Transforms/Transpilation/Layout.h"
+
+#include <utility>
+
+namespace mqt::ir::opt {
+
+/// @brief An Unit divides a quantum-classical program into routable sections.
```

Comment:
```suggestion
/// @brief A Unit divides a quantum-classical program into routable sections.
```
Might be worth checking through the PR for proper spelling of that.